Project-3/Project#3_tf

```diff
@@ -0,0 +1,292 @@
+ 
+Further, the transitivity property seems to be implied by the definition by defining the location and the target as aligned_with each other. Thus, if it can be shown that aligned_with should not have transitive listed as one of its properties, then the case for transitive not being listed as a property of located_in is further strenghtened. So, I add to my argument the case made against aligned_with by the group and discussed during group review. Now, one could argue that just because a property object like aligned_with is termed obsolete doesn't invalidate that it should have transitive listed as its property; if it weren't obsolete, it could have it listed without problem. Saying that a term is no longer used doesn't mean that it wasn't properly listed with transitive as one of its properties. To this objection, my response is the following. We may replace aligned_With with overlaps, which would seem to be its non-obsolete replacement/successor. This still does not give us good enough reasons to think that 'transitive' should be listed as a property for located_in. Simply because (a) target and (b) location overlap does not mean that the property located-in is transitive. The target will always be located-in the location that it is, but this just doubly asserts the same location fact. 
```

Comment:
Aligned with also strikes me as transitive, since it concerns alignment with spatial orientation :)